pages/vi/profiles/nandika-shendre.md

```diff
@@ -0,0 +1,8 @@
+# Nandika Shendre
+## Los Angeles, CA / PST Windows 10
+### I graduate in Dec 2017 from **University of Cincinnati.**
+### Main area of interest is DevOps.
+### I have experience in Python and Node JS.
+
+
+### LinkedIn URL: <www.linkedin.com/in/nandikashendre>
```

Comment:
Your LinkedIn link isn't visible. Github doesn't automatically hyperlink `<links in angle brackets>`. You should use `[this](format)` instead.